.storybook/components/token-table.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import Color from "@khanacademy/wonder-blocks-color";
+import {addStyle, View} from "@khanacademy/wonder-blocks-core";
+import {LabelMedium} from "@khanacademy/wonder-blocks-typography";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+
+const StyledTable = addStyle("table");
+const StyledTableRow = addStyle("tr");
+const StyledTableHeader = addStyle("th");
+const StyledTableCell = addStyle("td");
+
+type Column<T> = {
+    // The label to display in the table header.
+    label: string | React.ReactNode;
+    // The key to access the data in the row.
+    dataKey?: string;
+    // Custom cell renderer. If not provided, the value at `dataKey` will be
+    // used.
+    cell: string | ((row: T) => React.ReactNode);
+};
+
+type Props<T> = {
+    /**
+     * The data to render in the table.
+     * NOTE: We use generic types here to ensure that the data passed in is of
+     * the same type as the data in the rows.
+     */
+    data: Array<T>;
+    /**
+     * The columns to include in the table.
+     */
+    columns: Array<Column<T>>;
+};
+
+export default function TokenTable<T>({
+    columns,
+    data,
+}: Props<T>): React.ReactElement {
+    return (
+        <StyledTable style={styles.table}>
+            <thead>
+                <StyledTableRow style={styles.header}>
+                    {columns.map((column, i) => (
+                        <StyledTableHeader key={i} style={styles.cell}>
+                            {column.label}
+                        </StyledTableHeader>
+                    ))}
+                </StyledTableRow>
+            </thead>
+            <tbody>
+                {data.map((row, idx) => (
+                    <StyledTableRow key={idx} style={styles.row}>
+                        {columns.map((column, i) => (
+                            <StyledTableCell key={i} style={styles.cell}>
+                                {typeof column.cell === "string"
```

Comment:
**Optional suggestion:** I know it's explained in the type definition for `cell`, but do you think you could add a comment here too explaining why the cell type affects the usage of `row[column.cell]` vs `column.cell(row)`?